src/MusicPlaylist.java

```diff
@@ -0,0 +1,94 @@
+import components.queue.Queue;
+import components.queue.Queue1L;
+
+/**
+ * {@code MusicPlaylist} represented as {@Queue<String>} with implementations of
+ * primary methods.
+ *
+ * @convention <pre>
+ *[the key of $this.rep is the song title and the value of $this.rep is the artist of the song]
```

Comment:
This goes so far above and beyond the expectations for this assignment. Excellent work.